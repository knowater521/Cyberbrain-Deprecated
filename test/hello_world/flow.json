[
  {
    "code": "x = \"hello world\"",
    "next": "y = x",
    "prev": "",
    "step_into": "",
    "returned_from": ""
  },
  {
    "code": "y = x",
    "next": "cyberbrain.register(y)",
    "prev": "x = \"hello world\"",
    "step_into": "",
    "returned_from": ""
  },
  {
    "code": "cyberbrain.register(y)",
    "next": "",
    "prev": "y = x",
    "step_into": "",
    "returned_from": ""
  }
]