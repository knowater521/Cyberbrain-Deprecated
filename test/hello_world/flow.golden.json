[
  {
    "code": "x = \"hello world\"",
    "next": "y = x",
    "prev": "",
    "step_into": "",
    "returned_from": "",
    "var_changes": [
      "appear x=hello world\n"
    ]
  },
  {
    "code": "y = x",
    "next": "cyberbrain.register(y)",
    "prev": "x = \"hello world\"",
    "step_into": "",
    "returned_from": "",
    "var_changes": [
      "appear y=hello world\n"
    ],
    "tracking": [
      "x"
    ]
  },
  {
    "code": "cyberbrain.register(y)",
    "next": "",
    "prev": "y = x",
    "step_into": "",
    "returned_from": "",
    "tracking": [
      "y"
    ]
  }
]